### DISCUSSION TITLE
Shortcut in control panel

### DISCUSSION THREAD

**Discussion by plegrand1** [CatimaLoyalty/Android#1873] [General]
Hello,
I'm using a "Redmi Note 11 Pro 5G" with Android 13.
Since the Xiaomi update from "MIUI 14" to "Xiaomi HyperOS 1", a "Catima" shortcut has appeared in the control panel without me knowing why or how to remove it.
![image](https://github.com/CatimaLoyalty/Android/assets/8543044/f922ef44-da39-45f1-a7b4-fbe7dabb45f8)

What's more, when I click on this shortcut, I get this message:
![image](https://github.com/CatimaLoyalty/Android/assets/8543044/028d3800-7930-4baf-8ab2-bd1cb5da6c39)

Any idea how to solve this problem?
Thanks 
Pascal

**Comment by TheLastProject** [comment]
Ah, I see Xiaomi HyperOS 1 has put smart home control front-and-center! :)

Okay, so, this is a bit of a weird story, but long story short: your phone is suggesting using Catima to control your "smart home" due to how we worked around certain Google limitations. Allow me to elaborate:

When Google introduced the "Power menu" in Android 11 (a menu that shows all your cards and smart home devices when holding the power button), they limited it so that only Google Pay (formerly Google Wallet, now Google Wallet) could register cards to show there. However, one user really wanted their Catima cards easily reachable through the power menu for even faster check-outs at stores so discussed implementing this somehow in #146. Given Google blocks any app except Google Wallet from registering cards in the cards menu it was decided to instead make Catima tell Android it could control your smart home. Obviously, this isn't true, but it was the only way to put something in the "Power menu".

Now, it seems like Xiaomi put the smart home part of the "power menu" right there in your control panel in HyperOS. So, you could add some of your favourite Catima cards in there if you wanted for easier access.

I hope that answers your question as to why that shortcut is there.

Now for your question of how to remove it: I sadly can't confirm this as I don't own any Xiaomi device, but I did a quick web search for you and [according to this post](https://www.reddit.com/r/HyperOS/comments/1c2e69o/comment/kz9igb5) you should be able to replace/remove the Catima shortcut there by going to Settings -> Notifications & status bar and scrolling down to "Smart device controls" under "Control center" and choosing "None" or the name of a supported app you'd rather have appear there.

I'll move this to discussions as it's not really a bug, but expected behaviour given Google's limitations and more of a support request :)

**Reply by plegrand1** [reply]
Excellent!
Thank you for your very detailed reply. Now I have an explanation.
As for your solution to remove the shortcut, it works perfectly.
Thanks again.
